examples/typescript/src/gql/fragment-masking.ts

```diff
@@ -1,66 +1,86 @@
-import { ResultOf, DocumentTypeDecoration, TypedDocumentNode } from '@graphql-typed-document-node/core';
-import { FragmentDefinitionNode } from 'graphql';
-import { Incremental } from './graphql';
+/* eslint-disable */
+import {
+  ResultOf,
+  DocumentTypeDecoration,
+  TypedDocumentNode
+} from '@graphql-typed-document-node/core'
```

Comment:
wasn't this dependency supposed to be removed?

Reply:
this is not within the graphql-hooks package itself this is the example. we still want to allow 'typedDocumentNode'  files being called here but not have a dependency inside the package itself